sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -181,6 +181,13 @@ public abstract partial class Response<T>
         public static implicit operator T (Azure.Response<T> response) { throw null; }
         public override string ToString() { throw null; }
     }
+    public static partial class SerializationExtensions
+    {
+        public static System.BinaryData SerializeToBinaryData(this Azure.Core.Serialization.ObjectSerializer serializer, object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.ValueTask<System.BinaryData> SerializeToBinaryDataAsync(this Azure.Core.Serialization.ObjectSerializer serializer, object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Should these be instance methods? We can have a more optimized implementations in some cases.

Reply:
@KrzysztofCwalina @tg-msft thoughts?

Reply:
Oh.  Yeah.  They absolutely should.

Reply:
Agreed.